manifests/kustomize/third-party/argo/installs/namespace/cluster-scoped/kustomization.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
-- ../../../upstream/manifests/base/crds
+# Minimal CRDs omit schema validation, recommended for production cluster.
```

Comment:
to learn: what schema validation is referred to here?

Reply:
See https://github.com/argoproj/argo-workflows/tree/master/manifests/base/crds/full

> These CRDs have full schema validation. As a result, they are large and probably not suitable to be used in your cluster.

Reply:
Thanks!